src/Automation/TargetElementLocator.cs

```diff
@@ -23,7 +24,7 @@ public IEnumerable<A11yElement> LocateRootElements(int processId, IActionContext
             }
             catch (Exception ex)
             {
-                throw new AxeWindowsAutomationException(string.Format(CultureInfo.InvariantCulture, ErrorMessages.ErrorFailToGetRootElementsOfProcess, processId, ex), ex);
+                throw new AxeWindowsAutomationException(ErrorMessages.ErrorFailToGetRootElementsOfProcess.WithParameters(processId, ex), ex);
```

Comment:
Looking at this now, maybe we don't need to include the Exception in the message, since it's already in the innerException?